docs/conf.py

```diff
@@ -86,7 +90,7 @@ def __getattr__(cls, name):
 
 # General information about the project.
 project = u"TxPy"
-copyright = u"2016, Max Hodak, Tali Herzka, Jeremy Apthorp, Yang Choo, Peter Lee"
+copyright = u"2020, Strateos Inc."
```

Comment:
Do we need to do anything to actually make this copyright change? It might not be as simple as the other cases where we were just renaming the company. I'm assuming peter is ok with this but do we need to ask max/tali/jeremy?

Reply:
If they were all employees at the time of development we probably have a release that covers it, but to be on the safe side we might want them to sign a release once more? I’m not sure who to contact about setting that up

Reply:
hmm okay, Ill just update the date for now then